src/core/server/elasticsearch/version_check/ensure_es_version.test.ts

```diff
@@ -122,13 +144,13 @@ describe('pollEsNodesVersion', () => {
     internalClient.nodes.info.mockImplementationOnce(() => createEsError(error));
   };
 
-  it('returns iscCompatible=false and keeps polling when a poll request throws', (done) => {
+  it('returns isCompatible=false and keeps polling when a poll request throws', (done) => {
     expect.assertions(3);
     const expectedCompatibilityResults = [false, false, true];
     jest.clearAllMocks();
 
     nodeInfosSuccessOnce(createNodes('5.1.0', '5.2.0', '5.0.0'));
-    nodeInfosErrorOnce('mock request error');
+    nodeInfosErrorOnce(new Error('mock request error'));
```

Comment:
We _could_ test a plain string as an error but then we don't actually get any changes to the message portion itself. Before this work, we were effectively ignoring any errors thrown from the call to `nodes.info` and returning an object with a fake response:
```
catchError((_err) => {
  return of({ nodes: {} }); // ignore the error
})
```